proposals/lip-0063.md

```diff
@@ -218,50 +218,72 @@ ffffff
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `FACTOR_SELF_VOTES`                 | 10
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `BASE_VOTE_AMOUNT`                  | `10 * (10)^8`
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `MAX_LENGTH_NAME`                   | 20
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `MAX_NUMBER_SENT_VOTES`             | 10
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `MAX_NUMBER_PENDING_UNLOCKS`        | 20
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `FAIL_SAFE_MISSED_BLOCKS`           | 50
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `FAIL_SAFE_INACTIVE_WINDOW`         | 130,000
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `LOCKING_PERIOD_VOTES`              | 26,000
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `LOCKING_PERIOD_SELF_VOTES`         | 260,000
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `PUNISHMENT_WINDOW_VOTES`           | 260,000
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `PUNISHMENT_WINDOW_SELF_VOTES`      | 780,000
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `POM_LIMIT_BANNED`                  | 5
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `ROUND_LENGTH`                      | 103
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `BFT_THRESHOLD`                     | 68
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `MIN_WEIGHT`                        | 1000 x (10^8)
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `NUMBER_ACTIVE_DELEGATES`           | 101
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `NUMBER_STANDBY_DELEGATES`          | 2
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `TOKEN_ID_DPOS`                     | `TOKEN_ID_LSK = 0x 00 00 00 00 00 00 00 00`
```

Comment:
I would avoid the redundancy in the value definition here which will just create extra maintenance in case the value changes and put  `TOKEN_ID_LSK`.

Reply:
Good point. Changed